website/templates/include/profile/jobs.mako

```diff
@@ -17,20 +17,19 @@
 
                     <div class="well well-sm sort-handle">
                         <span>Position {{ $index() + 1 }}</span>
-                        <span data-bind="visible: $parent.hasMultiple()">
+                        <span data-bind="visible: $parent.contentsLength() > 1">
                             [ drag to reorder ]
```

Comment:
That 500 error was happening when I removed the observable contentsLength (which you asked to be removed).   The .length property was not available for the knockout variable at the right time.  hasMultiple was redundant however, so I deleted it and am using contentsLength instead, so we are still -1 knockout variable.  :facepunch: